### ISSUE TITLE
Version creation does not work

### ISSUE THREAD

**Issue by borodiychuk** [readmeio/rdme#206]
I have an issue with versions creation via the rdme client. Expected to have one more version after creation, but does not exist neither in listing, nor in the dashboard.
```
$ docker run -it -v /path/to/my/apidocs:/proj node:14-alpine /bin/sh
/ # export README_API_KEY=xxxxxxxxxxxxxxxxx
/ # npm install rdme -g
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
/usr/local/bin/rdme -> /usr/local/lib/node_modules/rdme/rdme.js
+ rdme@3.7.15
added 287 packages from 188 contributors in 37.552s
/ # rdme versions --key=$README_API_KEY
┌─────────┬──────────┬───────────────┬───────────┬─────────┬───────────┬──────────────────────────┐
│ Version │ Codename │ Is deprecated │ Is hidden │ Is beta │ Is stable │ Created on               │
├─────────┼──────────┼───────────────┼───────────┼─────────┼───────────┼──────────────────────────┤
│ 0       │ base     │ no            │ no        │ no      │ yes       │ 2020-06-16T13:22:32.017Z │
├─────────┼──────────┼───────────────┼───────────┼─────────┼───────────┼──────────────────────────┤
│ 1.0.0   │ base     │ no            │ no        │ no      │ no        │ 2020-06-16T13:22:32.017Z │
└─────────┴──────────┴───────────────┴───────────┴─────────┴───────────┴──────────────────────────┘
/ # rdme versions:create --version=1.0.1 --fork=1.0.0 --key=$README_API_KEY
/ # rdme versions --key=$README_API_KEY
┌─────────┬──────────┬───────────────┬───────────┬─────────┬───────────┬──────────────────────────┐
│ Version │ Codename │ Is deprecated │ Is hidden │ Is beta │ Is stable │ Created on               │
├─────────┼──────────┼───────────────┼───────────┼─────────┼───────────┼──────────────────────────┤
│ 0       │ base     │ no            │ no        │ no      │ yes       │ 2020-06-16T13:22:32.017Z │
├─────────┼──────────┼───────────────┼───────────┼─────────┼───────────┼──────────────────────────┤
│ 1.0.0   │ base     │ no            │ no        │ no      │ no        │ 2020-06-16T13:22:32.017Z │
└─────────┴──────────┴───────────────┴───────────┴─────────┴───────────┴──────────────────────────┘
/ # 
```
It works if I do a direct POST call to https://dash.readme.io/api/v1/version with curl.

**Comment by erunion** [comment]
I'm able to reproduce. Looking into it!

**Comment by erunion** [comment]
This is resolved under v3.7.16. Thanks for reporting it!
